### PR TITLE
feat(helm): update kube-prometheus-stack to 55.7.0

### PR DESCRIPTION
**Pull request by lippertmarkus**

### CODE REVIEW

**Comment by lippertmarkus** [comment]
@aantn could you please have a look? :)

**Comment by lippertmarkus** [comment]
@pavangudiwada  do you might have time to review? :)

**Comment by pavangudiwada** [comment]
We are sorry for the delay @lippertmarkus, our team is looking into this now

**Comment by pavangudiwada** [comment]
Update: @lippertmarkus  I'm working on this PR now, making some final checks for compatibility.

**Comment by lippertmarkus** [comment]
Would be great if we could add an automated process to keep the kube-prometheus-stack dependency up to date

File: docs/setup-robusta/upgrade.rst [hunk 1]
```diff
@@ -76,20 +76,24 @@ Manual upgrade instructions
...
+    kubectl replace -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.70.0/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml
+    kubectl replace -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.70.0/example/prometheus-operator-crd/monitoring.coreos.com_scrapeconfigs.yaml
+    kubectl replace -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.70.0/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+    kubectl replace -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.70.0/example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml
 
-3. Update Robusta with Helm. If you've lost ``generated_values.yaml``, you can :ref:`extract it from the cluster.<where-is-generated-values>`
+3. If you're upgrading from robusta v0.10.27 or below and using thanos please check if you need to adapt your config according to the `kube-prometheus-stack upgrade docs <https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/README.md#from-51x-to-52x>`.
```
Comment (aantn) [inline docs/setup-robusta/upgrade.rst:90]: Capital `R` in Robusta

Comment (lippertmarkus) [reply docs/setup-robusta/upgrade.rst]: fixed

**Comment by lippertmarkus** [comment]
@pavangudiwada @aantn did you have time to review? :)

**Comment by pavangudiwada** [comment]
Hi @lippertmarkus, extremely sorry,  this took way too long than expected. Thank you for your patience. We had to perform many checks to make sure the new version is compatible with the Robusta UI too. We hope this process will be faster next time.

For future reference, here are all the checks we performed:

1. **Install Robusta with updated Kube Prometheus Stack on new GKE and EKS clusters and check for issues.** Trigger demo alerts and see if they are working correctly.
    - Ensure there are no errors in pod logs for Prometheus Operator, Alertmanager, Prometheus, and Robusta Runner.
    - Make sure to check oomkilled pods, which should have graphs with data included.
    - Check at least one Prometheus alert (not an APIServer finding):
        - Before enabling managed Prometheus alerts in Robusta (the alert comes straight from KPS).
        - After enabling managed Prometheus alerts (the alert comes from our SaaS).
2. **Upgrade path:** Deploy the current version of Robusta without this fix and then deploy with the new KPS version. Look for any errors or issues.
3. **Ensure the functionality in the UI related to these specific pages still works after the updates.**
    - AI features.
    - Logs & Graphs on every page in Robusta, including:
        - Apps.
        - App details (a specific app).
        - Nodes.
        - Node details (a specific node).
        - Job details (for a job that ran more than 5 min).
4. **Silences** - Can create and view silences.
5. **Efficiency page** - Can run a new scan, view the results, and see recommendations (not a ‘?’ sign).
6. **Check the change log** for any new changes that we should be aware of.

**Review by aantn — APPROVED** [review]
LGTM, thank you

**Comment by lippertmarkus** [comment]
can we create a new release including this PR? :)

**Comment by aantn** [comment]
Included in 0.10.30!

**Pull request merged**